core/src/test/scala/unit/kafka/server/FetchSessionTest.scala

```diff
@@ -289,6 +289,94 @@ class FetchSessionTest {
     assertTrue(resp2.sessionId > 0)
   }
 
+  @Test
+  def testFetchManagerCacheIntegration(): Unit = {
```

Comment:
It seems like this is more of a test of correctly updating lastUsedMs, right?  "integration" is kind of vague (and also suggests that this is an integration test rather than unit test.)

Reply:
@cmccabe yeah, the name is bad and this is the second time I've changed it. I'll think of something else. I wanted to imply that it tests the interaction between the two, but you're right that there's a better name out there.